config/management/src/storage_helper.rs

```diff
@@ -120,6 +120,36 @@ impl StorageHelper {
         command.genesis()
     }
 
+    pub fn insert_waypoint(
+        &self,
+        remote_ns: &str,
+        waypoint_str: Option<&str>,
```

Comment:
these are mutually exclusive, but for storage helper let's only expose remote_ns

Reply:
sounds good!